lib/transforms/search-results-to-template-data.js

```diff
@@ -139,3 +141,33 @@ function createClearFilterLink (queryParams, rootUrl) {
   params['page[type]'] = queryParams.pageType;
   return `${rootUrl}/search${type}?${QueryString.stringify(params)}`;
 }
+
+function createClearFacetLink (queryParams, rootUrl) {
+  // queryParams.query contain already the query {"q":"ada","page[size]":50,"page[type]":"search"}
+  const links = {};
+  const type = queryParams.type !== 'all' ? '/' + queryParams.type : '';
+  const facetsName = queryParamsPerCategory[queryParams.type];
+  facetsName.forEach(function (facet) {
+    links[facet] = `${rootUrl}/search${type}?${QueryString.stringify(ignoreQueryParam(facet, queryParams.query))}`;
+  });
+  // special links for the date: need to ignore the both date at the same time
+  var queryPramsNoDates = ignoreQueryParam('filter[date[from]]', queryParams.query);
+  queryPramsNoDates = ignoreQueryParam('filter[date[to]]', queryPramsNoDates);
+  queryPramsNoDates = ignoreQueryParam('filter[birth[date]]', queryPramsNoDates);
+  queryPramsNoDates = ignoreQueryParam('filter[death[date]]', queryPramsNoDates);
```

Comment:
Could you make the params to ignore here an array? Then you'd only have to call this function once